akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2FrameProbe.scala

```diff
@@ -173,6 +175,13 @@ private[http] object Http2FrameProbe extends Matchers {
         (lastStreamId, ErrorCode.byId(reader.readIntBE()))
       }
 
+      override def expectSETTINGS(): Unit = {
+        // (6.5) The stream identifier for a SETTINGS frame MUST be zero (0x0).
+        // Ignore the payload for the time being. SETTINGS frames
+        // should have an empty payload anyway.
```

Comment:
I'm not sure I understand, aren't the settings part of the payload?

Reply:
I don't know what I was thinking when I wrote that. The last sentence is not correct.